src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,49 +1,104 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int LENGTH = 10;
+
+    private T [] defaultArray;
+    private int size;
+
+    public ArrayList() {
+        this.defaultArray = (T[]) new Object[LENGTH];
+        this.size = 0;
+    }
+
+    private boolean isFull() {
+        if (size >= defaultArray.length) {
+            T [] largeArray = (T []) new Object [(int)(defaultArray.length * 1.5)];
```

Comment:
```suggestion
            T[] largeArray = (T[]) new Object [(int) (defaultArray.length * 1.5)];
```
let's get used not to add whitespace between type and []?

Reply:
OK, I will try :)